test/inlineWrappedFunctions.test.ts

```diff
@@ -35,7 +35,7 @@ test('it can process nested calls of A2 with non identifiers as the first arg ',
   const { actual, expected } = transformCode(
     initialCode,
     expectedOutputCode,
-    createFunctionInlineTransformer(false, 'for tests')
+    createFunctionInlineTransformer(false, false, 'for tests')
```

Comment:
There was a missing argument for all of these. I just added `false`, but maybe we want some tests with `true` instead (though using `true` also makes the tests pass :man_shrugging: )

Reply:
Nice!  yeah, the tests are super old